packages/forms-web-app/src/views/macros/bullet-list.njk

```diff
@@ -0,0 +1,9 @@
+{% macro bulletList(points) %}
```

Comment:
We actually don't need this now as it is superseded by the work to allow additional html in AAPD-301